### DISCUSSION TITLE
As i heard that you are in partnership with Mozilla - Please do not adopt their minimalism addiction!

### DISCUSSION THREAD

**Discussion by Sa-Ja-Di** [Floorp-Projects/Floorp#292] [Q&A] [answered]
Hey, i really enjoy Floorp, but i have heard you are in a direct partnership with Mozilla.

And it is well known that Mozilla today hates customization, so i hope you are not going to allow Mozilla' anti-power user ideology to infect Floorp.

All i want to say is that i hope you plan to stay feature rich and not allow that Mozilla will dictate what for a concept you have to accept.

Thanks!

**Comment by Ryosuke-Asano** [comment, marked as answer]
I think you hear about connecting with Mozilla at gHack news comment.

This is a bit of a misunderstanding. There is no longer a Mozilla chapter in Japan. Instead, there is only WebDINO Japan, the former Mozilla Japan company.

It would be a lie to say that there is no relationship with this company at all, but it's not a big one, since they only receive opinions, so you don't need to worry about it. I love Mozilla, but I will not follow Firefox's policy at all, and Floorp will find its own way to be Floorp.

Please enjoy Floorp with peace of mind!

**Reply by Sa-Ja-Di** [reply]
Great to hear, that this blatant disrespect from Mozilla regarding the users which supported them from day one and only to have been abandoned and betrayed at version 28 is not coming over to Floorp.

Mozilla of old has to be respected. Period. But this new incarnation... makes it VERY hard to do so.

**Reply by Ryosuke-Asano** [reply]
We respect Mozilla because Floorp is based on Firefox, no matter how bad Mozilla is today. However, Floorp is going its own way.

Floorp was born while other derivatives were baffled by the XUL separation. A feature-rich Firefox-derived browser is what this world needs.
